### ISSUE TITLE
Progress bar behaving a bit weird

### ISSUE THREAD

**Issue by Jonasvsl** [nens/threedi-api-qgis-client#67]
* threedi-api-qgis-client version: 
1.5.0b
* Python version:
* Operating System:

### Description
![progres_bar](https://user-images.githubusercontent.com/1453159/85285262-e1837300-b490-11ea-9ac9-2a1f0ebab32c.PNG)

### What I Did

```
Paste the command(s) you ran and the output.
If there was a crash, please include the traceback here.
```

**Comment by Jonasvsl** [comment]
![image (2)](https://user-images.githubusercontent.com/1453159/85418944-c0df1a00-b571-11ea-98b8-f7ed1df07952.png)

"progress bar can be duplicated
this is occurs only with some simulations not started by the api client within qgis but they are probably caused by a mismatch on statusses of the simulations. for each simulation id there should be one bar showing the progress"

**Comment by DamnBack** [comment]
Fixed in the version 2.0.0
